src/main/java/com/minecolonies/coremod/entity/pathfinding/MinecoloniesAdvancedPathNavigate.java

```diff
@@ -742,55 +767,7 @@ public void updatePath() {}
     @Override
     protected void checkForStuck(@NotNull final Vec3d positionVec3)
```

Comment:
Can't we remove this then?

Reply:
no because that enables vanilla's logic for it